internal/xds/bootstrap/bootstrap.go

```diff
@@ -415,6 +418,15 @@ func NewConfigFromContents(data []byte) (*Config, error) {
 }
 
 func newConfigFromContents(data []byte) (*Config, error) {
+	// Normalize the input configuration, as this is used as the key in the map
```

Comment:
We are also doing a `json.Indent` which puts every json element in a newline. I feel the actual format of normalizing it not very important here. The fact is that we bring it to one format so that even if there is a slight whitespace difference in the config in tests, they will end up becoming the same config. As for non-test usage, this isn't much of deal because they will all use `NewConfig()` which simply reads from a single bootstrap file or env var.